src/traccuracy/matchers/_compute_overlap.py

```diff
@@ -9,6 +9,13 @@
 from skimage.measure import regionprops
 
 
+def _union_slice(a: tuple[slice], b: tuple[slice]):
```

Comment:
Python 3.8 is complaining about typing here - could we replace with 

```python
from typing import Tuple

def _union_slice(a: Tuple[slice], b: Tuple[slice]):
```

I think that should stop it from complaining.

Reply:
Thx! Should be fixed